.github/workflows/size-limit.yml

```
@@ -0,0 +1,38 @@
+name: Size Limit
```

Nit: My understanding is that this doesn't actually put a limit on the size right now, but rather simply reports changes in size. This might be a more apt name as a result:

```suggestion
name: Size Report
```